--- a/.vite/build/workspace-root-drop-handler-CSnqqPxa.js
+++ b/.vite/build/workspace-root-drop-handler-CSnqqPxa.js
@@ -38845,6 +38845,7 @@
     i = e.y(n.version);
   return (
     e.r({
+      enabled: !1,
       dsn: e.v,
       environment: Yk.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -65667,6 +65667,7 @@
   let n = oW(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (QA({
+    enabled: !1,
     dsn: aq,
     environment: e.buildFlavor,
     release: oq(n.version),
