--- a/.vite/build/workspace-root-drop-handler-Bom6Z7sW.js
+++ b/.vite/build/workspace-root-drop-handler-Bom6Z7sW.js
@@ -48232,6 +48232,7 @@
     i = e.L(n.version);
   return (
     e.r({
+      enabled: !1,
       dsn: e.I,
       environment: QF.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -74069,6 +74069,7 @@
   let n = QV(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (bk({
+    enabled: !1,
     dsn: aG,
     environment: e.buildFlavor,
     release: oG(n.version),
