--- a/webview/assets/chunk-LFPYN7LY-Cafl-tZa.js
+++ b/webview/assets/chunk-LFPYN7LY-Cafl-tZa.js
@@ -1058,11 +1058,16 @@
   unstable_useTransitions: o,
 }) {
   let s = r.useRef();
-  s.current ??= a({ initialEntries: n, initialIndex: i, v5Compat: !0 });
+  s.current ??= a({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: i,
+    v5Compat: !0,
+  });
   let c = s.current,
     [l, u] = r.useState({ action: c.action, location: c.location }),
     d = r.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         o === !1 ? u(e) : r.startTransition(() => u(e));
       },
       [o],
--- a/webview/assets/apps-ReuD7AbY.js
+++ b/webview/assets/apps-ReuD7AbY.js
@@ -2380,7 +2380,7 @@
   if (e == null) throw Error(`AppShellLayoutMotionContext is missing`);
   return e;
 }
-var va = P(I, !0),
+var va = P(I, window.__ELECTRON_SHIM__.initialSidebarState),
   ya = P(I, () => new Qt(1)),
   ba = P(L, !1),
   xa = P(L, () => new Qt(0)),
