--- a/webview/assets/settings-back-route-Cfgk-CZa.js
+++ b/webview/assets/settings-back-route-Cfgk-CZa.js
@@ -1,4 +1,5 @@
-import { t as e } from "./jsx-runtime-CiQ1k8xo.js";
+import { s as g } from "./chunk-Bj-mKKzh.js";
+import { n as _, t as e } from "./jsx-runtime-CiQ1k8xo.js";
 import { t } from "./clsx-DDuZWq6Y.js";
 import {
   S as n,
@@ -9,7 +10,8 @@
 } from "./setting-storage-EK1Te68s.js";
 import { _ as s, c } from "./app-shell-state-HP0T5lEX.js";
 import { t as l } from "./badge-DKQnFpmt.js";
-var u = o();
+var u = o(),
+  v = g(_());
 function d() {
   let e = (0, u.c)(3),
     t = r(n),
@@ -24,7 +26,12 @@
       (e[0] = t),
       (e[1] = a),
       (e[2] = o)),
-    i(`toggle-sidebar`, a, o));
+    i(`toggle-sidebar`, a, o),
+    v.useEffect(() => {
+      window.__ELECTRON_SHIM__.closeSidebar = () => {
+        s(t, !1);
+      };
+    }, [t]));
 }
 var f = e();
 function p(e) {
