--- a/webview/assets/chunk-LFPYN7LY-Cafl-tZa.js
+++ b/webview/assets/chunk-LFPYN7LY-Cafl-tZa.js
@@ -1058,11 +1058,16 @@
   unstable_useTransitions: o,
 }) {
   let s = r.useRef();
-  s.current ??= a({ initialEntries: n, initialIndex: i, v5Compat: !0 });
+  s.current ??= a({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: i,
+    v5Compat: !0,
+  });
   let c = s.current,
     [l, u] = r.useState({ action: c.action, location: c.location }),
     d = r.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         o === !1 ? u(e) : r.startTransition(() => u(e));
       },
       [o],
--- a/webview/assets/apps-Cel-9d-y.js
+++ b/webview/assets/apps-Cel-9d-y.js
@@ -2242,8 +2242,8 @@
   if (e == null) throw Error(`AppShellLayoutMotionContext is missing`);
   return e;
 }
-var ca = P(I, !0),
-  la = P(I, () => new Zt(1)),
+var ca = P(I, window.__ELECTRON_SHIM__.initialSidebarState),
+  la = P(I, () => new Zt(window.__ELECTRON_SHIM__.initialSidebarState)),
   ua = P(L, !1),
   da = P(L, () => new Zt(0)),
   fa = P(L, !1),
