--- a/webview/assets/chunk-LFPYN7LY-CkfOxD5s.js
+++ b/webview/assets/chunk-LFPYN7LY-CkfOxD5s.js
@@ -1058,11 +1058,16 @@
   unstable_useTransitions: o,
 }) {
   let s = r.useRef();
-  s.current ??= a({ initialEntries: n, initialIndex: i, v5Compat: !0 });
+  s.current ??= a({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: i,
+    v5Compat: !0,
+  });
   let c = s.current,
     [l, u] = r.useState({ action: c.action, location: c.location }),
     d = r.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         o === !1 ? u(e) : r.startTransition(() => u(e));
       },
       [o],
--- a/webview/assets/app-shell-state-HP0T5lEX.js
+++ b/webview/assets/app-shell-state-HP0T5lEX.js
@@ -543,8 +543,8 @@
   ut = ot.entries$,
   dt = { center: rt, left: it, right: at },
   ft = `app-shell-bottom-panel-launcher-visible`,
-  pt = c(o, !0),
-  mt = c(o, () => new g(1)),
+  pt = c(o, window.__ELECTRON_SHIM__.initialSidebarState),
+  mt = c(o, () => new g(window.__ELECTRON_SHIM__.initialSidebarState)),
   ht = t(ft, !0),
   gt = c(l, !1),
   _t = c(l, () => new g(0)),
